sormas-backend/src/main/java/de/symeda/sormas/backend/common/AbstractBaseEjb.java

```diff
@@ -42,13 +42,6 @@ protected AbstractBaseEjb(Class<ADO> adoClass, Class<DTO> dtoClass, SRV service,
 		this.userService = userService;
 	}
 
-	@Override
-	public DTO save(DTO dtoToSave) {
-		return save(dtoToSave, false);
-	}
-
-	// todo cannot be filled right now as we are missing ArchivableAbstractDomainObject
-	// with this abstract class e.g., ImmunizationFacadeEjb could be wired up to this as well
 	public abstract void archive(String uuid);
```

Comment:
Might be related: I think that `CoreAdo` should actually be what `DeletableAdo` is and we should have an `ArchivableAdo` instead which  `CoreAdo` extends. By this we, can hookup more code up to this in the future e.g., all the infrastructure code.

Reply:
I scrolled through the code if we do it I think there will be quite good wins w.r.t. code deduplication but this PR is already quite big. I think we go with Maté's remark and move it for now. We can do it later.

Reply:
the implementation of archive/dearchive is different for core and for infrastructure
but both have the same signature methods, in my opinion it would be enough to remove these when another abstraction comes below the base that has no archiving, but it is no trouble will remove it now

Reply:
@JonasCir CoreAdo is supposed to represent an entity that is related to a person, i.e., it has a semantic meaning that would be altered by your suggestion.